docs/guides/integrating-optimize.md

```diff
@@ -0,0 +1,121 @@
+---
+id: integrating-optimize
+title: Integrating Optimize
+sidebar_label: Integrating Optimize
+---
+
+<span class="badge badge--beginner">Beginner</span>
+<span class="badge badge--medium">Time estimate: 20 minutes</span>
+
+## Purpose
+
+The following document provides a basic end-to-end glance into Optimize and its features for new business users.
+
+Optimize offers business intelligence tooling for Camunda enterprise customers. By leveraging data collected during process execution, you can access reports, share process intelligence, analyze bottlenecks, and examine areas in business processes for improvement.
+
+With Optimize, review heatmap displays for instances which took longer than average to discover long-running flow nodes. As a result, reap actionable insights and rapidly identify the constraints of your system.
+
+See an in-depth overview of Optimize’s capabilities [here](https://docs.camunda.org/optimize/latest/).
+
+## Set up 
+
+A Camunda Cloud user can launch Optimize from the Cloud Console — the interface where users create clusters, and launch both Operate and Tasklist. Therefore, ensure you’ve [created a Camunda Cloud account](./getting-started/create-camunda-cloud-account.md) before getting started with Optimize for SaaS users.
```

Comment:
Lines 22 - 24 I can see we refer to "a user" instead of acknowledging the read as "you". Is this intentional?

Reply:
Nice catch on consistency here! Updated in latest commit.